### PR TITLE
[VL] Make glog config be session level

### PR DESCRIPTION
**Pull request by Yohahaha**
## What changes were proposed in this pull request?

After this patch, user can change glog config and debug config in one session more easily.

### CODE REVIEW

File: cpp/velox/compute/VeloxBackend.cc [hunk 1]
```diff
@@ -148,12 +142,14 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
...
-    uint32_t severityLogLevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
-    FLAGS_minloglevel = severityLogLevel;
+    FLAGS_v = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
+    FLAGS_minloglevel = veloxcfg->get<uint32_t>(kGlogSeverityLevel, kGlogSeverityLevelDefault);
   } else {
-    FLAGS_v = 99;
+    if (veloxcfg->isValueExists(kGlogVerboseLevel)) {
+      FLAGS_v = veloxcfg->get<uint32_t>(kGlogVerboseLevel, kGlogVerboseLevelDefault);
+    } else {
+      FLAGS_v = 99;
```
Comment (marin-ma) [inline cpp/velox/compute/VeloxBackend.cc:151]: It should be better to define kGlogVerboseLevelMaximum and replace the "99" with it.

Comment (Yohahaha) [reply cpp/velox/compute/VeloxBackend.cc]: ok.

**Comment by Yohahaha** [comment]
@marin-ma @zhztheplayer @PHILO-HE would you help review? thanks!

**Review by PHILO-HE — APPROVED** [review]
Looks good!

**Comment by zhztheplayer** [comment]
Thanks!

**Pull request merged**